File: FSWO102-JV/lesson08/handsOnL08/Dog.java
package handsOnL08;

public class Dog implements Mammal {

	@Override
	public void speak() {
		System.out.println("Bark!");

	}

	@Override
	public void run() {
		System.out.println("Dogs can run at a top speed of 45 mph!");

	}

	@Override
	public void eat() {
		System.out.println("Dogs eat bones.");
		
	}
	
	public static void main(String[] args) {
//		Dog bowwow = new Dog();
		Mammal bowwow = new Dog();
		bowwow.speak();
		bowwow.run();
		bowwow.eat();

	}



}

File: FSWO102-CS/Lesson08/HandsOnL08/HandsOnL08/Classes/Dog.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using HandsOnL08.Interfaces;

namespace HandsOnL08.Classes
{
    public class Dog : IMammal
    {
        public void Eat()
        {
            Console.WriteLine("Dogs eat bones.");
        }

        public void Run()
        {
            Console.WriteLine("Dogs can run at a top speed of 45 mph!");
        }

        public void Speak()
        {
            Console.WriteLine("Bark!");
        }
    }
}
